tests/helpers.py

```diff
@@ -14,9 +15,16 @@ def getQs(self: Qs) -> Dict[str, str]:
         return dict(
             map(
                 lambda x: x.split("="),
-                self._mock.call_args.args[0][0].split("?")[1].split("&"),
+                self._mock.call_args.args[0][0].split("?")[1].split("&")
             )
         )
+    
+    def getPOQs(self: Qs) -> Dict[str, str]:
```

Comment:
Remove the unquote is ok to be done, the main reason for having the dedicated check method is because the "self._mock.call_args" that is used to retrieve the API link built, works differently between "Query" and "GmePO". 

I'm not sure where it comes from but in the case of "Query" it is populated with a list of lists and consequently, the API data to be checked are taken as "self._mock.call_args.args[0][0]", in case of "GmePO" instead is just a list, and the API data to be checked are taken as "self._mock.call_args.args[0]" 

The only difference I've found is in the _exec method in the 2 class:
def _exec(self: GMEPublicOfferQuery, url: str) -> Any:
def _exec(self: _Query, urls: List[str]) -> list:

but aligning it doesn't change the outcome of "self._mock.call_args", Do you know @AndreaCuneo which method is mocking that may cause the difference in the returned data?